plugins/micromega/vect.mli

```diff
@@ -22,6 +23,9 @@ type t (** The type of vectors or equivalently linear expressions.
            Moreover, the representation is spare and variables with a zero coefficient
            are not represented.
         *)
+type t
```

Comment:
Why are two aliased types defined here?

Reply:
Because otherwise, t gets shadowed in the module Bound module (end of vect.mli)